tests/qgis-projects/tests/treeview.qgs.cfg

```diff
@@ -1,16 +1,15 @@
 {
     "metadata": {
         "qgis_desktop_version": 32815,
-        "lizmap_plugin_version_str": "4.3.5-alpha",
-        "lizmap_plugin_version": 40305,
-        "lizmap_web_client_target_version": 30800,
-        "lizmap_web_client_target_status": "Dev",
-        "instance_target_url": "http://localhost:8130/",
-        "instance_target_repository": "testsrepository"
+        "lizmap_plugin_version_str": "4.3.3",
```

Comment:
You should update your plugin.

Reply:
https://github.com/3liz/lizmap-web-client/pull/4426#discussion_r1599479717